### ISSUE TITLE
Option to show descriptions under title as opposed to summary

### ISSUE THREAD

**Issue by Monklite** [razonyang/hugo-theme-bootstrap#140]
Some people use the description parameter in their posts to display under the title in list sections; however, the current configuration is ti show the first (I think) 280 characters of it instead. This definitely has its place, though I think an option to choose would be nice. I can do a PR for this if it is something you want though it's not much work, so it's fine if you would rather do it.  

What are your thoughts?

**Comment by khannzxc** [comment]
the description is mainly used for search engines.

**Comment by Monklite** [comment]
Interesting. My only gripe with the current implementation (likely a limitation of Hugo) is that most of my posts start with `# Introduction` as a header, so the description will read like `Introduction Lorem Ipsum...` so I wonder if there's a way to omit headers or if that's out of the scope given the design of Hugo

**Comment by razonyang** [comment]
@Monklite This theme uses the post summary(end by `<!--more-->`), length up to 320.

```markdown
Here is summary...
<!--more-->
```

But the summary will also appears in the post page. Does this suit your requirements?

This is a reasonable request if you only want to display the summary in the list, not in the article.

**Comment by Monklite** [comment]
> @Monklite This theme uses the post summary(end by `<!--more-->`), length up to 320.
> 
> ```gfm
> Here is summary...
> 
> ```
> 
> But the summary will also appears in the post page. Does this suit your requirements?
> 
> This is a reasonable request if you only want to display the summary in the list, not in the article.

My problem with the current implementation (which isn't really an issue with the theme itself, rather Hugo I'm pretty sure) is that I see header name content in the summary due to how `.Summary` works. Usually I have a parameter in each post in the front matter called `description` and whatever I fill in there is what shows under the title in lists. I hope that makes sense

**Comment by razonyang** [comment]
@Monklite I see, I'll handle it later.

**Comment by Monklite** [comment]
Appreciate it!

**Comment by razonyang** [comment]
@Monklite Fixed by 3606d65aa4ae99a095a460cea4e7dec1300d157b. It introduces two parameters: 

```
[post]
  # excerpt = "description"
  # excerptMaxLength = 120
```
